Lib/test/crashers/null_code_obj.py

```diff
@@ -0,0 +1,24 @@
+import platform
+
+"""
+This execute a payload code object which crashes the interpeter for different
+python versions. Currently there is no bytecode verifier to prevent those
+crashes related to malformed bytecode
+"""
+
+def crash_38():
+    exec((lambda f: f).__code__.__class__(0,0,0,0,0,0,b"\x01",(),(),(),"","",0,b""))
+
+
+def crash():
+    locals()['crash_38'] = crash_38
+    locals()['crash_39'] = crash_38
+    locals()['crash_310'] = crash_38
+
+    py_ver = "".join(platform.python_version_tuple()[0:2])
+    crash_func = "crash_{}".format(py_ver)
+    locals()[crash_func]()
```

Comment:
You should remove all of the tests for the version number, and just call the code directly here. It just needs to be a one liner.

Reply:
Alright, it's done as expected 👍

Reply:
Hi, are we good for merging it now? Let me know if anything else is needed 🤝

Reply:
There's a discussion on the bug tracker issue discussing whether this crasher is needed, given what's already present.